packages/@react-stately/table/src/utils.ts

```diff
@@ -0,0 +1,113 @@
+import {GridNode} from '@react-types/grid';
+import {Key} from 'react';
+
+type mappedColumn<T> = GridNode<T> & {
+    index: number,
+    delta: number,
+    calculatedWidth?: number
+};
+
+export function getContentWidth(widths: Map<Key, number>): number {
+  return Array.from(widths).map(e => e[1]).reduce((acc, cur) => acc + cur, 0);
+}
+
+// numbers and percents are considered static. *fr units or a lack of units are considered dynamic.
+export function isStatic(width: number | string): boolean {
+  return width != null && (!isNaN(width as number) || (String(width)).match(/^(\d+)(?=%$)/) !== null);
+} 
+
+function parseFractionalUnit(width: string): number {
+  if (!width) {
+    return 1;
+  } 
+  let match = width.match(/^(\d+)(?=fr$)/);
+      // if width is the incorrect format, just deafult it to a 1fr
+  if (!match) {
+    console.warn(`width: ${width} is not a supported format, width should be a number (ex. 150), percentage (ex. '50%') or fr unit (ex. '2fr')`, 
+        'defaulting to \'1fr\'');
+    return 1;
+  }
+  return parseInt(match[0], 10);
+}
+
+export function parseStaticWidth(width: number | string, tableWidth: number): number {
+  if (typeof width === 'string') {
+    let match = width.match(/^(\d+)(?=%$)/);
+    if (!match) {
+      throw new Error('Only percentages or numbers are supported for static column widths');
+    }
+    return tableWidth * (parseInt(match[0], 10) / 100);
+  }
+  return width;
+}
+    
+    
+export function getMaxWidth(maxWidth: number | string, tableWidth: number): number {
+  return maxWidth != null
+        ? parseStaticWidth(maxWidth, tableWidth)
+        : Infinity;
+}
+
+export function getMinWidth(minWidth: number | string, tableWidth: number): number {
+  return minWidth != null
+      ? parseStaticWidth(minWidth, tableWidth)
+      : 75;
```

Comment:
magic number? where is this coming from?

Reply:
75 is the pre-existing min. https://github.com/adobe/react-spectrum/blob/main/packages/%40react-stately/layout/src/TableLayout.ts#L97

Reply:
We should probably pull it out into a shared variable so we know this
it's either that or it needs to be sent in from the Layout since not every Layout will have this as the default min width

Reply:
Danny clarified, it was moved, it doesn't exist in two places. So I'm ok with this